src/discussions/navigation/breadcrumb-menu/BreadcrumbMenu.jsx

```diff
@@ -1,66 +1,91 @@
-    },
-  ].filter(crumb => Boolean(crumb.label))
-    .map(({
-      url,
-      label,
-    }) => ({
-      url: url(),
-      label,
-    }));
-
-  const activeLabel = crumbs.pop().label;
+  const showAllMsg = intl.formatMessage(messages.showAll);
 
   return (
     <div className="breadcrumb-menu d-flex flex-row mt-2 mx-3">
-      <Breadcrumb
-        links={crumbs}
-        activeLabel={activeLabel}
-        clickHandler={(e) => {
-          e.preventDefault();
-          history.push(e.target.pathname);
-        }}
-      />
+      <DropdownButton
+        title={blocks[currentChapter]?.displayName || showAllMsg}
+        variant="outline"
+        onSelect={(cId) => dispatch(setCurrentChapter(cId))}
+      >
+        <Dropdown.Item eventKey={null} key="null" active={currentChapter === null}>
+          {showAllMsg}
+        </Dropdown.Item>
+        {chapters.map(chapter => (
+          chapter.topics.length > 0
+          && (
+            <Dropdown.Item eventKey={chapter.id} key={chapter.id} active={chapter.id === currentChapter}>
+              {chapter.displayName}
+            </Dropdown.Item>
+          )
+        ))}
+      </DropdownButton>
+      {currentChapter
+        && (
+          <>
+            <div className="d-flex py-2">/</div>
+            <DropdownButton
+              title={blocks[currentSequential]?.displayName || showAllMsg}
+              variant="outline"
+              onSelect={(sId) => dispatch(setCurrentSequential(sId))}
+            >
+              <Dropdown.Item eventKey={null} key="null" active={currentSequential === null}>
+                {showAllMsg}
+              </Dropdown.Item>
+              {blocks[currentChapter].children.map(seqId => (
+                blocks[seqId].topics.length > 0
+                && (
+                  <Dropdown.Item eventKey={seqId} key={seqId} active={seqId === currentSequential}>
+                    {blocks[seqId].displayName}
+                  </Dropdown.Item>
+                )
+              ))}
+            </DropdownButton>
+          </>
+        )}
+      {currentSequential
+        && (
+          <>
+            <div className="d-flex py-2">/</div>
+            <DropdownButton
+              title={blocks[currentVertical]?.displayName || showAllMsg}
+              variant="outline"
+              onSelect={(vId) => dispatch(setCurrentVertical(vId))}
+            >
+              <Dropdown.Item eventKey={null} key="null" active={currentVertical === null}>
+                {showAllMsg}
+              </Dropdown.Item>
+              {blocks[currentSequential]?.children?.map(vertId => (
+                blocks[vertId].topics.length > 0
+                && (
+                  <Dropdown.Item eventKey={vertId} key={vertId} active={vertId === currentVertical}>
+                    {blocks[vertId].displayName}
+                  </Dropdown.Item>
+                )
+              ))}
+            </DropdownButton>
+          </>
+        )}
     </div>
```

Comment:
The version will be used and developed further when the new provider's topic are further developed.

Reply:
Alternatively, I can remove this code right now and add it back in when the new topics are implemented.

Reply:
I think we can leave it as is add a todo Item for future reference, so that we don't have to remove and add it back when needed?